integration/ksm/ksm_test.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -e
+
+dir_path=$(dirname "$0")
+source "${dir_path}/../../lib/common.bash"
+source "${dir_path}/../../metrics/lib/common.bash"
+
+TEST_NAME="Run containers with ksm"
+KATA_KSM_THROTTLER="${KATA_KSM_THROTTLER:-yes}"
+RUNTIME="${RUNTIME:-kata-runtime}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+IMAGE="${IMAGE:-busybox}"
+WAIT_TIME="60"
+
+FIRST_TMP_FILE=$(mktemp firstcontainer.XXXX)
+SECOND_TMP_FILE=$(mktemp secondcontainer.XXXX)
+
+function remove_tmp_files() {
+	rm -rf $FIRST_TMP_FILE $SECOND_TMP_FILE
+}
+
+trap remove_tmp_files EXIT
+
+function turn_on_ksm() {
+	echo "Turning on KSM"
+	echo "1" > ${KSM_ENABLE_FILE}
```

Comment:
This assumes the script is running as `root`, but there is no check on that in `run_with_ksm()`.